page.php

```diff
@@ -1,3 +1,11 @@
+<?php 
+
+/* 
+ *		Template Name: Custom Page
+ *		Description: No Sidebar
+ */
+
+?>
```

Comment:
Like I explained on friday, your page.php is your general page template. You use the page.php template for all general pages, if you need a specific one for ie. the contact page ONLY then you create a custom page template.